modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/latex_catsuit.dm

```diff
@@ -95,3 +95,10 @@
 	breasts_overlay.icon_state = "breasts"
 	breasts_icon_overlay = mutable_appearance('modular_skyrat/modules/modular_items/lewd_items/icons/obj/lewd_clothing/lewd_uniform.dmi', "none")
 	breasts_icon_overlay.icon_state = "breasts"
+
+/obj/item/clothing/under/misc/retro/skimpy/retro/suit
```

Comment:
Is there a reason for this to be skimpy/retro/suit, or could it just be skimpy/retro? (In other words, is skimpy/retro an entirely unused/undefined/nonexistant item)